src/Renderer/ThreeExtented/FeatureToolBox.js

```diff
@@ -0,0 +1,559 @@
+/**
+ * Generated On: 2016-09-28
+ * Class: FeatureToolBox
+ * Description:
+ */
+
+import THREE 					from 'THREE';
+import CVML 					from 'Core/Math/CVML';
```

Comment:
I need CVML for triangulation, the module have exit in master so i reused it. 

We can use THREE.Js for extrusion, it may work but doing extrusion by our traditional way, it's simpler to add styled building (see roof's color and facade's color) that is not possible with THREEJS extrusion.

Otherwise, i prefer that we create an other branch to improve it because this request may take many time.